src/player.cpp

```diff
@@ -5169,31 +5124,15 @@ void player::suffer()
                     done_effect = true;
                 }
                 // Shaking
-                if( !done_effect && one_turn_in( 4_hours ) ) {
+                if( !done_effect && !has_effect( effect_valium ) && one_turn_in( 4_hours ) ) {
                     add_msg( m_bad, _( "You start to shake uncontrollably." ) );
                     add_effect( effect_shakes, rng( 2_minutes, 5_minutes ) );
                     done_effect = true;
                 }
                 // Shout
                 if( !done_effect && one_turn_in( 4_hours ) ) {
-                    std::vector<std::string> shouts{ _( "\"Get away from there!\"" ),
-                                                     _( "\"What do you think you're doing?\"" ),
-                                                     _( "\"Stop laughing at me!\"" ),
-                                                     _( "\"Don't point that thing at me!\"" ),
-                                                     _( "\"Stay away from me!\"" ),
-                                                     _( "\"No! Stop!\"" ),
-                                                     _( "\"Get the fuck away from me!\"" ),
-                                                     _( "\"That's not true!\"" ),
-                                                     _( "\"What do you want from me?\"" ),
-                                                     _( "\"I didn't mean to do it!\"" ),
-                                                     _( "\"It wasn't my fault!\"" ),
-                                                     _( "\"I had to do it!\"" ),
-                                                     _( "\"They made me do it!\"" ),
-                                                     _( "\"What are you!?\"" ),
-                                                     _( "\"I should never have trusted you!\"" ) };
-
-                    std::string i_shout = random_entry_ref( shouts );
-                    shout( "yourself shout, " + i_shout );
+                    std::string snip = SNIPPET.random_from_category( "schizo_self_shout" );
+                    shout( "yourself shout, " + snip );
```

Comment:
```suggestion
                    shout( string_format( _( "yourself shout, %s" ), snip ) );
```
The `_()` translates the text (it was previously not translated at all).

And the use of `stringformat` with "%s" makes the text better to translate (the translation can now move the string from `snip` around as it may be required by the grammar of some languages).

Reply:
Ah, I was wondering why that one was different.